akd_client/src/lib.rs

```diff
@@ -44,184 +34,20 @@
 //! cargo build --no-default-features --features sha3_256
 //! ```
 //!
-//! ### Additional features
-//!
-//! Additionally there are some features **not** related to the underlying hash function utilization
-//!
-//! 1. _wasm_: Compile with web-assembly support for WASM compilation
-//! 2. _wee_alloc_: Utilize the WEE allocator, which is roughly 1KB instead of 10KB as a allocator but slower. This
-//! is _helpful_ in cases of constrained binary footprint size to help minimize
-//! 3. _nostd_: Disable use of the std library
-//! 4. _vrf_: Enable verification of VRFs client-side. Requires addition of the crates [`curve25519-dalek`] and [`ed25519-dalek`]
-//! as dependencies
-//!
-//! You can compile and pack the WASM output with
-//! ```bash
-//! cd akd_client # optional
-//! wasm-pack build --features wasm
-//! ```
-//! which currently has a resultant WASM file size of ~191KB with VRF verification enabled
-//!
-//! #### WASM Compilation and Deployment
-//!
-//! For WASM deployment of the AKD client, you'll want to read the [wasm_bindgen](https://rustwasm.github.io/wasm-bindgen/reference/deployment.html)
-//! documentation which has reference material dependent on your environment.
-//!
-//! # Client Types
-//!
-//! A small note about the types in this library. They are specifically independent of the main AKD crate because
-//! it's assumed that to perform a verification at the edge, the client will have had to receive some over-the-air
-//! message which contains the data inside the proof. Therefore they'd need to be deserialized and handled independently
-//! of the AKD crate which wouldn't be a dependency anyways. This is why the types are independent and specified separately
-//! from the core AKD types.
-//!
+
 #![warn(missing_docs)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(feature = "nostd", no_std)]
 extern crate alloc;
-#[cfg(feature = "nostd")]
-use alloc::string::String;
-#[cfg(feature = "wasm")]
-use wasm_bindgen::prelude::*;
-// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
-// allocator.
-#[cfg(feature = "wee_alloc")]
-#[global_allocator]
-static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-#[cfg(not(feature = "nostd"))]
-use std::fmt::Display;
-
-// Types are re-exported at the root level for visbility
-mod types;
-pub use types::*;
-// verify types are not re-exported, to not clutter the root path
-pub mod verify;
+// Re-expose the core functionality from akd_core (verifications, types, etc)
+pub use akd_core::verify;
+pub use akd_core::*;
 
 #[cfg(feature = "protobuf")]
```

Comment:
```suggestion
#[cfg(feature = "protobuf_serialization")]
```

Reply:
Hmm actually `protobuf` should work as well, there's an implicit feature defined by the `protobuf` dependency. Feel free to ignore this comment

Reply:
<3